### ISSUE TITLE
train(val=False) still performs validation after training?

### ISSUE THREAD

**Issue by nikolaydyankov** [ultralytics/ultralytics#1968]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm not sure if this is a bug, or am I misunderstanding the docs, so I'm posting it as a question.

This is how I'm training:

```
model.train(data=data_yaml, epochs=epochs, name=experiment_path + '/train', val=False, exist_ok=True)
```

My goal is to get the validation metrics for each individual image, that is why I'm calling `model.val()` separately with `save_json`:

```
model.val(name=val_results_dir, save_json=True)
```

Unless there is a way to get the metrics from the train function, I would like to disable the automatic validation after the training. However, the `val=False` argument doesn't work. Is this a bug?

**Comment by nikolaydyankov** [comment]
As a side question, is `save_json=True` the correct way to get the metrics for each image?

**Comment by nikolaydyankov** [comment]
What I'm trying to do is get the worst performing masks and the images where those masks are.

**Comment by glenn-jocher** [comment]
@AyushExel improved val introspection request, i.e. see worst performing images.

**Comment by pderrenger** [comment]
@nikolaydyankov that sounds like a valuable tool! Your contribution would indeed benefit the community. It's great that you're bringing a different skillset to the table. If you have any specific questions or need guidance while working on the UI, feel free to ask here or in the discussions. Your input is much appreciated!
